data/categories.txt

```diff
@@ -822,7 +822,7 @@ pl:^Sklep z elektroniką|sklep elektroniczny
 pt:^Loja de telemóveis|loja de aparelhos eletrónicos
 pt-BR:^Loja de celulares|telefone|compras
 ro:^Magazin de telefoane mobile|magazin de electronice
-es:^Celulares|Tienda de telefonía móvil|tienda de electrónica
+es:^Móviles|Tienda de telefonía móvil|tienda de aparatos electrónicos
```

Comment:
Are you sure that "tienda de aparatos electrónicos" is relevant here?
This shop is specifically for mobile phones, after all.

Reply:
In this case I have translated "electronics store" from "en"